src/cpu/o3/store_set.hh

```diff
@@ -61,21 +123,52 @@ struct ltseqnum
  */
 class StoreSet
 {
+  protected:
+  std::string _name;
+
   public:
-    typedef unsigned SSID;
+    typedef Addr SSID;
 
   public:
+    std::string name() const { return _name; }
+
+    class SSITEntry : public CacheEntry
+    {
+        SSID _ssid;
+      public:
+        using IndexingPolicy = gem5::SSITIndexingPolicy;
```

Comment:
I don´t think we need all this overhead, as both classes behave the same. I think this suffices:
```
class SSITEntry : public CacheEntry
{
    SSID _ssid;
public:
    void setSSID(SSID id) { _ssid = id; }
    SSID getSSID(void) const { return _ssid; }
};
```

Reply:
Done.